commands.go

```diff
@@ -61,7 +61,7 @@ type statefulCmdable struct {
 
 //------------------------------------------------------------------------------
 
-func (c statefulCmdable) Auth(password string) *StatusCmd {
+func (c *statefulCmdable) Auth(password string) *StatusCmd {
```

Comment:
There is also the worry that a call to `func (c statefulCmdable) Auth(password string) *StatusCmd` would copy the whole `Pipeline/Client` struct.  I cannot confirm it, but that's what it looks like in https://godoc.org/gopkg.in/redis.v4#Pipeline.Auth